src/cmap/connection.ts

```diff
@@ -562,6 +562,15 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
       callback(err);
     }
   }
+
+  moreToComeCommand(
```

Comment:
Are we attached to the name `moreToComeCommand`?  The publicly documented many-to-one behavior is referrred to as "exhaust" (exhaust allowed, exhaust cursors, etc), and `moreToCome` is just an implementation detail of exhaust support.   `exhaustCommand` might make more sense

Reply:
SGTM